src/stage.c

```diff
@@ -607,6 +610,7 @@ stage_request(struct view *view, enum request request, struct line *line)
 			return REQ_NONE;
 		}
 		stage_split_chunk(view, line);
+		reset_search(view);
```

Comment:
Looks obviously correct. I guess we could do this on successful splits only, but I don't see a good reason to favor one over the other

```diff
diff --git a/src/stage.c b/src/stage.c
index 1b0a92e2..0e869381 100644
--- a/src/stage.c
+++ b/src/stage.c
@@ -404,6 +404,7 @@ stage_split_chunk(struct view *view, struct line *chunk_start)
 	if (chunks) {
 		stage_insert_chunk(view, &header, chunk_start, NULL, NULL);
 		redraw_view(view);
+		reset_search(view);
 		report("Split the chunk in %d", chunks + 1);
 	} else {
 		report("The chunk cannot be split");
```